resources/definitions/flsun_sr.def.json

```
@@ -0,0 +1,76 @@
+{
+    "version": 2,
+    "name": "FlSun SuperRacer",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Thushan Fernando",
+        "manufacturer": "Flsun",
+        "platform": "flsun_sr.3mf",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "has_machine_quality": false,
+        "preferred_quality_type": "fast",
+        "machine_extruder_trains": {
+            "0": "flsun_sr_extruder_0"
+        }
+    },
+    "overrides": {
+        "machine_extruder_count": {
+            "default_value": 1
+        },
+        "retraction_enable": {
+            "default_value": true
+        },
+        "machine_heated_bed": {
+            "default_value": true
+        },
+        "machine_width": {
+            "default_value": 260
+        },
+        "machine_depth": {
+            "default_value": 260
+        },
+        "machine_height": {
+            "default_value": 330
+        },
+        "machine_center_is_zero": {
+            "default_value": true
+        },
+        "machine_head_with_fans_polygon": {
+            "default_value": [
+                [0, 0],
+                [0, 0],
+                [0, 0],
+                [0, 0]
+            ]
+        },
+        "z_seam_type": {
+            "default_value": "back"
```

```suggestion
            "default_value": "'back'"
```
I missed that in the previous review. The enumaration values should be inside single quotes (in addition to the double outside quotes) to be recognized as enums.

---

Updated with thanks.